src/screens/UserManagement/UserRevamp/UserUtils.res

```diff
@@ -0,0 +1,149 @@
+let moduleListRecoil: Recoil.recoilAtom<array<UserManagementTypes.userModuleType>> = Recoil.atom(
```

Comment:
why atom is defined over here?

Reply:
As this atom is only used for the user management module

Reply:
Moved the atom to HyperswitchAtom